### DISCUSSION TITLE
Make FA a variable font, like Material

### DISCUSSION THREAD

**Discussion by jfbrennan** [FortAwesome/Font-Awesome#19862] [Feature Requests]
Is Font Awesome working on a variable font version? If not, is that in the roadmap? If not, why not?

**Comment by sensibleworld** [comment]
It's on our radar, yes. Lots of considerations, and not entirely sure we'd do it for our entire library... but we're interested in it!

**Comment by kurtrank** [comment]
I would really love to see an option for a variable font! Material Icons does this and it is super nice, however their icon names (ligatures) are sometimes a bit odd.
